tests/unit_tests/test_channels_first_last.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+
+import tensorflow as tf
+from src.models import model_name_to_class
+
+class TestCompareChannelsFirstLast(unittest.TestCase):
+  @classmethod
+  def setUpClass(cls):
+    cls.kwargs = {
```

Comment:
name of this variable is too generic. Maybe name it 'model_kwargs' ?

Reply:
updated accordingly.